---------------------
PatchSet 329 
Date: 2010/10/02 13:58:35
Author: millert
Branch: HEAD
Tag: (none) 
Log:
Revert rev 1.13, TIOCGSIZE is actually older that TIOCGWINSZ.

Members: 
	pod/perlfaq8.pod:1.13->1.14 

Index: src/gnu/usr.bin/perl/pod/perlfaq8.pod
diff -u src/gnu/usr.bin/perl/pod/perlfaq8.pod:1.13 src/gnu/usr.bin/perl/pod/perlfaq8.pod:1.14
--- pod/perlfaq8.pod:1.13	Sat Oct  2 19:48:12 2010
+++ pod/perlfaq8.pod	Sat Oct  2 20:58:35 2010
@@ -231,10 +231,10 @@
 illustrative:
 
 	require 'sys/ioctl.ph';
-	die "no TIOCGSIZE " unless defined &TIOCGSIZE;
+	die "no TIOCGWINSZ " unless defined &TIOCGWINSZ;
 	open(TTY, "+</dev/tty")                     or die "No tty: $!";
-	unless (ioctl(TTY, &TIOCGSIZE, $winsize='')) {
-		die sprintf "$0: ioctl TIOCGSIZE (%08x: $!)\n", &TIOCGSIZE;
+	unless (ioctl(TTY, &TIOCGWINSZ, $winsize='')) {
+		die sprintf "$0: ioctl TIOCGWINSZ (%08x: $!)\n", &TIOCGWINSZ;
 	}
 	($row, $col, $xpixel, $ypixel) = unpack('S4', $winsize);
 	print "(row,col) = ($row,$col)";
